banyand/kv/kv.go

```diff
@@ -126,6 +126,17 @@ func TSSWithFlushCallback(callback func()) TimeSeriesOptions {
 	}
 }
 
+func TSSWithMemTableSize(size int64) TimeSeriesOptions {
+	return func(store TimeSeriesStore) {
+		if size < 1 {
```

Comment:
Do we have some minimal requirement for the mem table?

Reply:
No, we don't limit any values to more than 1 bit.